src/main/scala/ru/d4team/chat/api/PersonController.scala

```diff
@@ -0,0 +1,21 @@
+package ru.d4team.chat.api
+
+import ru.d4team.chat.services.PersonService
+import zio._
+import zio.json._
+import zio.http._
+import zio.http.model.Method
+
+trait PersonController {
+  def route: HttpApp[Any, Nothing]
+}
+
+final class PersonControllerImpl(personService: PersonService) extends PersonController {
+  private def getAll: UIO[Response] = personService.getAll.map(seq => Response.json(seq.toJson))
+
+  override def route: HttpApp[Any, Nothing] = Http.collectZIO[Request] { case Method.GET -> !! / "persons" => getAll }
```

Comment:
After refactoring service we end up with getAll of `Task[Response]` type.

Ideally, we have to log an error and provide a meaningful error message for user if something happend.
As for now, let's just simulate this behaviour is a simple form.

For example, it might looks like this:
```scala
trait PersonController {
  def route: HttpApp[Any, Nothing]
}

object PersonController {
  val live: URLayer[PersonService, PersonController] = ZLayer.fromFunction(PersonControllerImpl.apply _)
}

final case class PersonControllerImpl(personService: PersonService) extends PersonController {
  private def getAll: Task[Response] = personService.getAll.map(seq => Response.json(seq.toJson))

  override def route: HttpApp[Any, Nothing] =
    Http.collectZIO[Request] { case Method.GET -> !! / "persons" =>
      getAll.tapErrorCause(ZIO.logCause).orElseFail(new RuntimeException("Something go wrong"))
    }
}
```

Reply:
Decided to return a response with the error

Reply:
Will handle the ADT errors with statuses after we implement them